### ISSUE TITLE
Need Help! Loss nan

### ISSUE THREAD

**Issue by Allenhu47** [summerlvsong/Aggregation-Cross-Entropy#2]
**for this line: torch.log(input)**

The 'input' is the softmax score (0-1).
If **k**-th class does not show in an input, the accumulative softmax score of all time steps for k-th class is very likely to be 0. Then this will result into torch.log(input) = nan.

How do you make sure that 'input' does not equal to 0 for 'torch.log(input)'

**Comment by summerlvsong** [comment]
NAN problem fixed.  Please refer to line 34 of 'source/models/seq_module.py'.

**Comment by allen4747** [comment]
> NAN problem fixed. Please refer to line 34 of 'source/models/seq_module.py'.

Thanks!

**Comment by TangDL** [comment]
make sure input > 0
